DXRando/DeusEx/Classes/DXRLamp2.uc

```diff
@@ -1,17 +1,60 @@
 //=============================================================================
-// Lamp2 except Warm White.  Fuck daylight bulbs.
+// Lamp2 except with a warm white light at the top.
+// Fuck daylight bulbs.
 // Localization files overwrite the ItemName,
 // so really force it to not be halogen
 //=============================================================================
 class DXRLamp2 injects #var(prefix)Lamp2;
 
+var DynamicLight lt;
+
+function InitLight()
+{
+    local vector loc;
+
+    if (lt == None) {
+        lt = Spawn(class'DynamicLight');
+
+        loc = Location;
+        loc.z += CollisionHeight * 1.35;
+        lt.SetLocation(loc);
+
+        lt.LightHue=44;
+        lt.LightSaturation = 160;
+        lt.LightBrightness = 255;
+        // increasing the radius to 25 causes weird flickering on a couch in the Free Clinic, probably elsewhere
+        lt.LightRadius = 20;
```

Comment:
I felt like 20 was too much, especially in Alex's office

```suggestion
        lt.LightRadius = 15;
```

Reply:
20 still seems pretty dark to me. There's plenty of dark area in Alex's office, even just in between the two lamps. I was inclined to bump it up to 25, before I discovered that it causes glitches.

On the other hand, the game is a bit strangely dark in general so maybe lower is appropriate. I'm personally still on the fence, I guess.